API/api/models.py

```
@@ -1,3 +1,93 @@
+    end_time = models.TimeField()
+    slot_duration_minutes = models.PositiveSmallIntegerField()
+    break_start_time = models.TimeField(blank=True, null=True)
+    break_end_time = models.TimeField(blank=True, null=True)
+    is_active = models.BooleanField(default=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        db_table = "plages"
+        indexes = [
+            models.Index(fields=["event", "name"]),
+        ]
```

Ruff (0.14.14)

[warning] Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)